library/ZendService/Twitter/Response.php

```diff
@@ -43,6 +43,12 @@ class Response
     protected $rawBody;
 
     /**
+     * @var array
+     */
+    protected $rateLimit;
```

Comment:
`$rateLimit` seems to have a specific structure. Consider converting it into a tiny value object rather than using arrays